### DISCUSSION TITLE
I don't see iRacing in the setup

### DISCUSSION THREAD

**Discussion by luchan123** [SeriousOldMan/Simulator-Controller#21] [General]
I can see my AC, ACC and AMS but no iRacing. How would I add it?

**Comment by luchan123** [comment]
Answered my own questions after reading the guides. It is iRacingUI.exe that you have to locate, that is under the iRacing install folder in the UI folder. :)

**Comment by SeriousOldMan** [comment]
Great, you found a way forward. If you have anything else (yes, it is a complex piece of software), let me know. Best on Discord.
